signature-specification.md

```diff
@@ -120,125 +120,105 @@ Notary v2 requires the signature envelope to support the following signed attrib
 
 ### Supported Signature Envelopes
 
-#### JWS JSON Serialization
+#### COSE_Sign1_Tagged
 
-In JWS JSON Serialization ([RFC7515](https://datatracker.ietf.org/doc/html/rfc7515)), data is stored as either claims or headers (protected and unprotected).
-Notary v2 uses JWS JSON Serialization for the signature envelope with some additional constraints on the structure of claims and headers.
+In COSE ([rfc8152](https://datatracker.ietf.org/doc/html/rfc8152)), data is stored as either payload or headers (protected and unprotected).
+Notary v2 uses [COSE_Sign1_Tagged](https://datatracker.ietf.org/doc/html/rfc8152#section-4.2) object as the signature envelope with some additional constraints on the header fields.
 
 Unless explicitly specified as OPTIONAL, all fields are required.
-Also, there shouldn’t be any additional fields other than ones specified in JWSPayload, ProtectedHeader, and UnprotectedHeader.
 
-**JWSPayload a.k.a. Claims**:
-Notary v2 is using one private claim (`notary`) and two public claims (`iat` and `exp`).
-An example of the claim is described below
+**Payload**: COSE signs the payload as defined in the [Payload](#payload) section.
 
-```json
+**ProtectedHeaders**: Notary v2 supports only the following protected headers:
```

Comment:
I would suggest clarifying that other headers can be included, but Notary v2 will not process them (and will fail if they are set as 'crit'). For example, some COSE implementations will always add 'alg' - it's fine if nv2 won't rely on this, but I don't believe it should be a failure if it is present.